utils/server/websocket.go

```diff
@@ -152,3 +164,189 @@ func (s *Server) handleWebsocket() http.HandlerFunc {
+							case <-ticker.C:
+								channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_CONNECTION_KEEP_ALIVE}}
+							}
+						}
+					}()
+				}
+
+			case utils.GQL_START:
+				data := new(model.RealtimeRequest)
+
+				parserSource := source.NewSource(&source.Source{
+					Body: []byte(m.Payload.Query),
+				})
+				// parse the source
+				doc, err := parser.Parse(parser.ParseParams{Source: parserSource})
+				if err != nil {
+					channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_CONNECTION_ERROR, Error: []gqlError{{Message: err.Error()}}}}
+					closeConnAliveRoutine <- true
+					return
+				}
+
+				opDefinition, ok := doc.Definitions[0].(*ast.OperationDefinition)
+				if !ok {
+					channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_CONNECTION_ERROR, Error: []gqlError{{Message:errors.New("erros in operation definition of schema").Error()}}}}
+					closeConnAliveRoutine <- true
+					return
+				}
+
+				v, ok := opDefinition.SelectionSet.Selections[0].(*ast.Field)
+				if !ok {
+					channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_CONNECTION_ERROR, Error:[]gqlError{{Message: errors.New("error in selection set of schema").Error()}}}}
+					closeConnAliveRoutine <- true
+					return
+				}
+
+				for _, dirValue := range v.Arguments {
+					if dirValue.Name.Value == "skipInitial" {
+						if dirValue.Value.(*ast.BooleanValue).Value {
+							data.Options = model.LiveQueryOptions{SkipInitial: true}
+						}
+					}
+				}
+
+				whereData, err := graphql.ExtractWhereClause(v.Arguments, utils.M{})
+				if err != nil {
+					channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_ERROR, Error: []gqlError{{Message: err.Error()}}}}
+					closeConnAliveRoutine <- true
+					return
+				}
+				data.Where = whereData
+				data.Token = token
+				dbType, err := graphql.GetDBType(v)
+				if err != nil {
+					channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_ERROR, Error: []gqlError{{Message: err.Error()}}}}
+					closeConnAliveRoutine <- true
+					return
+				}
+				data.DBType = dbType
+				data.Project = project
+				data.Group = v.Name.Value
+				data.Type = m.Type
+				data.ID = m.ID
+				graphqlIDMapper.Store(m.ID, data.Group)
+
+				// Subscribe to realtime feed
+				feedData, err := s.realtime.Subscribe(ctx, clientID, s.auth, s.crud, data, func(feed *model.FeedData) {
+					channel <- &graphqlMessage{ID: feed.QueryID, Payload: payloadObject{Status: utils.GQL_DATA, Data: graphqlSuccessData{Type: feed.Type, Doc: feed.Payload, DocID: feed.DocID}}}
+				})
+
+				if err != nil {
+					channel <- &graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_ERROR, Error: []gqlError{{Message: err.Error()}}}}
+					closeConnAliveRoutine <- true
+					return
+				}
+
+				channel <- (&graphqlMessage{ID: m.ID, Payload: payloadObject{Status: utils.GQL_START, Data: feedData}})
```

Comment:
you can't just pass feedData like this. You'll have to iterate over it and make a `graphqlSuccessData` for each one of them and pass that in the channel in the loop itself